### PR TITLE
Observer antaghud use is extended to 5 minutes after roundstart

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

This extends the #23009 PR to allow people observing the game within 5 minutes of roundstart (not just before the game has actually started) to antaghud without losing respawnability. 

## Why It's Good For The Game
Sometimes players leave their computers between rounds, and when they come back the round has already started. If they want to observe in this case, I think they should have a chance to have the same benefits as of just before the round.

## Testing

Observed 2 minutes after the game started, was a roundstart observer
set the config to 0 minutes
observed 2 minutes after the game started, was not a roundstart observer
restarted, observed before the game started, was a roundstart observer

## Changelog
:cl:
tweak: Observing before 5 minutes into a round, lets you toggle AntagHUD without losing respawnability.
/:cl:

### CODE REVIEW

File: code/modules/mob/new_player/new_player.dm
Comment (lewcc) [inline code/modules/mob/new_player/new_player.dm:191]:
```suggestion
				var/period_human_readable = "within [GLOB.configuration.general.roundstart_observer_period] minute\s"
```

**Review by lewcc — COMMENTED** [review]
One nit. Looks good beyond this, but will need aa to handle the config update.

**Review by lewcc — APPROVED** [review]
LGTM, pending AA review and team approval

**Review by S34NW — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]
CFG change committed

**Pull request merged**
